albumentations/augmentations/geometric/resize.py

```diff
@@ -51,7 +54,7 @@ class LongestMaxSize(DualTransform):
     """Rescale an image so that maximum side is equal to max_size, keeping the aspect ratio of the initial image.
 
     Args:
-        max_size (int): maximum size of the image after the transformation.
+        max_size (int, list of int): maximum size of the image after the transformation.
```

Comment:
Could you please add descriptions about how transform would workd with `list` parameter. Add information that the values will be selected from a list, and not randomly sampled from interval.

Reply:
Updated docstring